packages/slice-machine/tests/project/cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
same for these files, I don't think they make sense here

Reply:
Plugins?, 
We'll probably end-up needing them at some point.